configure

```diff
@@ -213,6 +213,11 @@ parser.add_option('--with-dtrace',
     dest='with_dtrace',
     help='build with DTrace (default is true on sunos)')
 
+parser.add_option('--with-lttng',
+    action='store_true',
+    dest='with_lttng',
+    help='build with Lttng (Only available to Linux)')
```

Comment:
Can it default to 'on' on Linux, or  will that break Linuxen that don't have LTTNG?